pvlib/tools.py

```diff
@@ -277,51 +277,57 @@ def _build_args(keys, input_dict, dict_name):
 
 # Created April,2014
 # Author: Rob Andrews, Calama Consulting
-
-def _golden_sect_DataFrame(params, VL, VH, func):
+# Modified: November, 2020 by C. W. Hansen, to add atol and change exit
+# criteria
+def _golden_sect_DataFrame(params, VL, VH, func, atol=1e-8):
     """
     Vectorized golden section search for finding MPP from a dataframe
     timeseries.
 
     Parameters
     ----------
-    params : dict
-        Dictionary containing scalars or arrays
+    params : dict or Dataframe
+        Parameters for the IV curve(s) where MPP will be found. Must contain
+        keys: 'r_sh', 'r_s', 'nNsVth', 'i_0', 'i_l'
         of inputs to the function to be optimized.
         Each row should represent an independent optimization.
```

Comment:
Should these two lines be removed?  It doesn't flow perfectly with the new text

Reply:
I rewrote that parameter description, most of that text is leftover from a previous version.